ci/test-04-options-a-b.pl

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/perl -w
 
-use Test::Command tests => 32;
+use Test::Command tests => 35;
```

Comment:
The tests are counted differently: you have added 9 tests (each of the added `exit_is_num()`, `stdout_like()`, `stderr_is_eq()` is counted as one test). Please adjust this.

Reply:
I thought that each command counted as a test. Thank you for your explanation! I changed to `41` (`32 + 9`).